proto/v1/instance_service.proto

```diff
@@ -324,14 +326,16 @@ message DataSourceExternalSecret {
     // ref: https://developer.hashicorp.com/vault/docs/auth/token
     TOKEN = 1;
     // ref: https://developer.hashicorp.com/vault/docs/auth/approle
-    APP_ROLE = 2;
+    VAULT_APP_ROLE = 2;
     // ref: https://aws.github.io/aws-sdk-go-v2/docs/configuring-sdk/#specifying-credentials
```

Comment:
I'm wondering if we can simplify it a bit. The SecretType for AWS and GCP doesn't need this AuthType. Or at least we can add some comments about what this AuthType does.

Reply:
Agreed, I'd like to remove the `AWS_ENVIRONMENT` and `GCP_SERVICE_ACCOUNT`